Description: Support reproducible builds
 Stripping __DATE__ and __TIME__ helps make the builds reproducible.
 
 See also the configure-stamp target in debian/rules where MKREPRO_DATE and
 MKREPRO_TIME are set.
Forwarded: not-needed
Origin: vendor
Author: Richard Laager <rlaager@wiktel.com>
Last-Update: 2016-11-29
--- a/wafhelpers/waf.py
+++ b/wafhelpers/waf.py
@@ -36,7 +36,7 @@
 
 	def run(self):
 		self.outputs[0].write("""
-const char *Version = "%s " __DATE__ " " __TIME__;
+const char *Version = "%s";
 const char *VVersion = "CFLAGS=%s LDFLAGS=%s";
 """ % (self.env.NTPSEC_VERSION_STRING, "Need-CFLAGS", "Need-LDFLAGS"))
 
